packages/SwingSet/src/kernel/vat-warehouse.js

```diff
@@ -107,7 +108,9 @@ export function makeSyscallSimulator(
   let replayError; // sticky
 
   const explain = () => {
-    console.log(`anachrophobia strikes ${vatID} on delivery ${deliveryNum}`);
+    console.log(
+      `anachrophobia strikes ${vatID} syscalls for delivery ${deliveryNum}`,
```

Comment:
Grammar nit, maybe "anachrophobia strikes vatID for syscalls on delivery deliveryNum"? It's the vat that's at fault, and it faulted during a delivery.